packages/changed-elements-react/src/widgets/comparisonJobWidget/components/VersionCompareManageNamedVersions.tsx

```diff
@@ -0,0 +1,19 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import "./styles/ComparisonJobWidget.scss";
+
+interface ManageNamedVersionsProps {
+  children: React.ReactNode;
```

Comment:
Add `import type { ReactNode } from "react";` import statement and then use `ReactNode`. I worry consumers may not have `React` in their global type space.